tests/data_context/store/test_store_backends.py

```diff
@@ -1342,6 +1356,8 @@ def test_TupleS3StoreBackend_list_over_1000_keys():
     assert len(keys) == num_keys_to_add + 1
 
 
+@pytest.mark.cloud
+@pytest.mark.unit
```

Comment:
nope! I think that string is just for test anyways it's just a random UUID - will update accordingly

Reply:
so that access token is actually used in `assert_called_with` calls - it is referenced in `shared_called_with_request_kwargs` so it's necessary here. I can decouple these deps but maybe in a separate PR?

Reply:
My understanding is we are trying to be stringent when applying the `unittest` marker since these likely won't be revisited. So a followup PR which decouples this or a comment that says something to the effect of "we don't care what this string is but it is used to match the asserts" would be fine. I just want to be clear to the reader that this doesn't need to be a valid cloud token since we are not hitting cloud directly here.